src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax;
+
+import core.basesyntax.db.dao.StorageDao;
+import core.basesyntax.db.dao.StorageDaoImp;
+import core.basesyntax.model.GoodsOperation;
+import core.basesyntax.service.CsvParseService;
+import core.basesyntax.service.FileReadService;
+import core.basesyntax.service.FileWriteService;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.imp.CsvParser;
+import core.basesyntax.service.imp.FileReadServiceImp;
+import core.basesyntax.service.imp.FileWriteServiceImp;
+import core.basesyntax.service.imp.Reporter;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationsStrategy;
+import core.basesyntax.strategy.handlers.BalanceOperationHandler;
+import core.basesyntax.strategy.handlers.PurchaseOperationHandler;
+import core.basesyntax.strategy.handlers.ReturnOperationHandler;
+import core.basesyntax.strategy.handlers.SupplyOperationHandler;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+    private static final String NAME_SOURCE_FILE = "SourceFile.txt";
+    private static final String PATH_SOURCE_FILE = "src/main/resources/" + NAME_SOURCE_FILE;
+    private static final String NAME_CONSUMER_FILE = "ConsumerFile.txt";
+    private static final String PATH_CONSUMER_FILE = "src/main/resources/" + NAME_CONSUMER_FILE;
+    private static final Map<String, Integer> storage;
+    private static final FileReadService fileReadService;
+    private static final CsvParseService csvParseService;
+    private static final Map<GoodsOperation.TransactionType, OperationHandler> strategies;
+    private static final StorageDao storageDaoImp;
```

Comment:
minor: name should be just storageDao, we suggest that there could be multiple implementations and we don't know which exactly will be used here